TranslatorReasonerAPI.yaml

```diff
@@ -208,10 +215,16 @@ components:
         - id
     EdgeBinding:
       type: object
+      description: >-
+        A instance of EdgeBinding is a single KnowledgeGraph Edge mapping,
+        identified by the corresponding 'id' object key identifier of the
+        Edge within the Knowledge Graph. Future iterations of the EdgeBinding
+        record may include other annotation, such as a score to qualify the
```

Comment:
Other annotations like these are currently allowed since `additionalProperties` defaults to `true`. They are just not currently standardized.

Reply:
Can we clarify this?

Reply:
Should these annotations all be captured in the node and edge attributes in the knowledge graph? Or are there use cases where they make more sense in the bindings?

Reply:
One item that we'd like to add as an annotation on a binding is a score - How well does this kedge align with this qedge?

Reply:
@patrickkwang, I did mention the `score` as the example of extra binding data. I have been taking this PR as mainly a secondary documentation PR but I guess as long as `score` is not currently specified as `required`, I guess there is no harm in adding it (documented) into this PR, right?